### PR TITLE
GA Console Polish -- Latest Design

### PR DESCRIPTION
**Pull request by georgevanderson**
Sorry the PR is so big. This is the work to align the console with Alex's latest designs.

Main thing to note is I am linking to the models page from all the infrastructure objects, so their pages are not visitable. I kept the code in their in case we want to revert.

Most of the PR is updating styles to match the figma and combining similar code. Some of which was tracked in linear
Linear tickets:
https://linear.app/514/issue/514-498/python-dataframes-snippet-for-consumption
https://linear.app/514/issue/514-438/web-add-favicon-to-docs
https://linear.app/514/issue/514-533/web-datamodels-usage-tab-has-overflowing-text-in-code-snippet
https://linear.app/514/issue/514-439/web-consistent-font-styles
https://linear.app/514/issue/514-440/mobile-moosejs-docs-logo-off-center
https://linear.app/514/issue/514-426/enable-browser-next-and-previous-page-navigation-for-tabs
https://linear.app/514/issue/514-443/web-card-styles-on-overview

### CODE REVIEW

**Review by callicles — APPROVED** [review]
Love the abstraction work into components.
Nice work!

File: apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx [hunk 1]
```diff
@@ -64,19 +65,19 @@ export default async function Page({
 
   return (
     <section className="p-4 max-h-screen flex-grow overflow-y-auto flex flex-col">
-      <div className="py-10">
-        <div className="text-6xl">
-          <Link className="text-muted-foreground" href={"/"}>
-            ../
+      <div className="text-base text-muted-foreground flex">
+        <Fragment>
```
Comment (tg339) [inline apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx:52]: Any reason you're using fragment here vs just <></>? Mainly a curiosity thing

File: apps/moose-console/src/lib/snippets.ts [hunk 2]
```diff
@@ -103,20 +103,15 @@ export const curlSnippet = (data: CliData, model: DataModel) => {
...
+    http://${data.project && data.project.local_webserver_config.host}:${data.project.local_webserver_config.port}/${ingestionPoint.route_path}`;
 };
 
 export const bashSnippet = (data: CliData, model: DataModel) => {
   const curlCommand = curlSnippet(data, model);
 
-  return `
-  #!/bin/bash
-
+  return `#!/bin/bash
```
Comment (tg339) [inline apps/moose-console/src/lib/snippets.ts]: Looking like you have some indent issues here. Recommend newlining at the top and then making it easy to scan the snippets by sticking them all to the leftmost indent

**Review by tg339 — APPROVED** [review]
Looking great. Indent issue still present and one curiosity comment

File: apps/moose-console/src/lib/snippets.ts (hunk 2 above)
Comment (tg339) [reply apps/moose-console/src/lib/snippets.ts]: I fixed the indents and some of the snippets in my PR if you rebase

File: apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx (hunk 1 above)
Comment (georgevanderson) [reply apps/moose-console/src/app/infrastructure/databases/[databaseName]/tables/[tableId]/page.tsx:52]: Honestly totally forgot it was a thing. Think we used to have a linting rule about it? So just habit

File: apps/moose-console/src/lib/snippets.ts (hunk 2 above)
Comment (georgevanderson) [reply apps/moose-console/src/lib/snippets.ts]: New lining adds a new line to the displayed code block. 
image

Opted for adding a backslash. A lil weird but also you to scan 
image

**Pull request merged**